### ISSUE TITLE
Tkinter overhaul

### ISSUE THREAD

**Issue by friedrichrieber** [pluto-development-team/puck-tracker-pluto#9]
making the tkinter gui scalabe and better to look at

**Comment by zerbian** [comment]
I would propose what change the design to have a forward and backward Button and not individual tabs.

**Comment by friedrichrieber** [comment]
I agree, makes the software easier to understand and is probably the better way.

And then at the end something like this, right ? 
<img width="391" alt="skizze" src="https://user-images.githubusercontent.com/31022596/45698225-d1c25680-bb67-11e8-96ee-3f52867b5a7e.png">
